jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/RangeSlider.java

```diff
@@ -132,14 +135,14 @@ public void setRightValue(int valueRight, boolean isVertical) {
     }
 
     @JDIAction("Set slider from '{name}' to {0}, {1}")
-    public void setValue(int valueLeft, int valueRight, boolean isVertical) {
-        List<Double> nowValue = getCurrentLocationValue();
-        if (isVertical) {
+    public void setValue(int valueLeft, int valueRight) {
+        List<Integer> nowValue = getValue();
```

Comment:
If I understand correctly, the value can be double. Because there is a `step` parameter in which we can specify a double value too.